public/js/selfoss-ui.js

```diff
@@ -176,13 +176,13 @@ selfoss.ui = {
     entrySelect: function(entry) {
         if (selfoss.ui.selectedEntry !== null) {
             selfoss.ui.selectedEntry.removeClass('selected');
-            selfoss.ui.selectedEntry.attr('aria-current', 'false');
+        $('.entry-title > .entry-title-link', entry).attr('aria-current', 'false');
```

Comment:
At least this one should refer to the old `selectedEntry`. `entry` is the one we want to select. 
```suggestion
            $('.entry-title > .entry-title-link', selfoss.ui.selectedEntry).attr('aria-current', 'false');
```

Reply:
> At least this one should refer to the old `selectedEntry`. `entry` is the one we want to select.

I've done the changes, but I don't think so that it is working appropriately. Now, even though I'm collapsing the entry, Screen reader is still announcing 'current' for the entry. it mean's that aria-current="false" is not invoking. Having said that, aria-current is only going away from the previous entry if I'm expanding another entry. and again, same is happening with it as well after collapsing. aria-current is still true.

Reply:
> At least this one should refer to the old `selectedEntry`. `entry` is the one we want to select.

I feel that 'selfoss.ui.selectedEntry' is not needed for aria-current

Reply:
> Now, even though I'm collapsing the entry, Screen reader is still announcing 'current' for the entry.

That is correct behaviour. The last entry you interacted with will be selected (visually, there is a darker border around it). Entry being selected means you can use selfoss keyboard shorcuts to expand it again (<kbd>o</kbd>), open the link (<kbd>v</kbd>), or switch to a previous or next entry (<kbd>j</kbd> and <kbd>k</kbd>, or arrow keys).